pkg/apis/messaging/v1/in_memory_channel_defaults.go

```diff
@@ -35,9 +37,10 @@ func (imc *InMemoryChannel) SetDefaults(ctx context.Context) {
 		imc.Annotations[messaging.SubscribableDuckVersionAnnotation] = "v1"
 	}
 
+	ctx = apis.WithinParent(ctx, imc.ObjectMeta)
 	imc.Spec.SetDefaults(ctx)
 }
 
 func (imcs *InMemoryChannelSpec) SetDefaults(ctx context.Context) {
-	// TODO: Nothing to default here...
+	imcs.Delivery.SetDefaults(ctx)
```

Comment:
@pierDipi should we do that on other downstream channels too?

Reply:
Original issue came up in here: https://github.com/knative-sandbox/eventing-kafka-broker/pull/1254

Not sure if it is done in other places too

Reply:
Sure, I'll do it.